scripts/run_upp

```diff
@@ -35,7 +39,7 @@ set -x
 #----------------------------------------------------------------------------------
```

Comment:
Link on L38 for the UPP Usres Guide does not work - let's get this updated

Reply:
That works too. I think the only place latest is linked is on the wiki and points to develop currently.